collectors/schema.yaml

```diff
@@ -0,0 +1,210 @@
+type: "object"
+properties:
+  name:
+    type: "string"
+  title:
+    type: "string"
+  duplicate_for_virtual_integrations:
+    type: "object"
+    properties:
+      list:
+        type: "array"
+        items:
+          type: "string"
+  overview:
+    type: "object"
+    properties:
+      how_the_collector_works:
+        type: "object"
+        properties:
+          description:
+            type: "string"
+      supported_platforms:
+        type: "object"
+        properties:
+          description:
+            type: "string"
+      multi-instance:
+        type: "boolean"
+      permissions:
+        type: "object"
+        properties:
+          description:
+            type: "string"
+  setup:
+    type: "object"
+    properties:
+      prerequisites:
+        type: "object"
+        properties:
+          list:
+            type: "array"
+            items:
+              type: "object"
+              properties:
+                title:
+                  type: "string"
+                text:
+                  type: "string"
+      behavior:
```

Comment:
I think 
- this should be `default_behavior`.
- and be a part of the Overview. Why do you think it is Setup?

Reply:
Agree for name,

The doc had it there, so I went with that

Reply:
Ok, but what are your thoughts? Is default behavior Setup?

Reply:
Actually I agree that this is overview, I'm with you on this, there is no configuration for it, it just gives out info

Reply:
I also see that configuration is outside of setup, I will move that under setup too